app/views/groups/_member.html.erb

```diff
@@ -0,0 +1,21 @@
+<li itemscope itemtype="http://schema.org/Code">
+  <div class="header">
+    <div class="header-content">
+      <%= link_to member.user, rel: 'member', itemprop: 'username' do %><%= member.user.username %><% end %>
+      <div class="remove">
+        <% if admin_member?(current_user, member.group) %>
+          <% if member.admin? %>
+            <% if admin_members(member.group).count > 1 %>
```

Comment:
Only show remove button for an admin member when there is more than one admin member of the group.